data/scenario_patches/aead175.parkpatch

```diff
@@ -0,0 +1,19 @@
+{
+    "scenario_name": "Botany Breakers",
+    "sha256": "aead175fbe37f3c98321080482689e30092cde7474de7266b01d288656cc516d",
+    "water": [
+        {
+            "height": 96,
+            "coordinates": [
+                [39, 50], [39, 51], [39, 52], [39, 53], [39, 54], [39, 55], 
+                [40, 51], [40, 52], [40, 53], [40, 54], 
+                [41, 52], [41, 53], [41, 54], [41, 55], 
+                [42, 51], [42, 52], [42, 53], [42, 54], [42, 55], 
+                [43, 50], [43, 51], [43, 52], [43, 53], [43, 54], [43, 55], 
+                [44, 50], [44, 51], [44, 52], [44, 54], [44, 55], [44, 56], 
+                [45, 55], [45, 56], [45, 57], 
+                [46, 55], [46, 56], [46, 57]
```

Comment:
Please update this one to use the same indenting as the "Botany Breakers (.sea)" file (`2ca68df.parkpatch`)

Reply:
Thanks for rebasing, I have updated the indenting of the Botany Breakers parkpatch.